### PR TITLE
WRP-5664: Add UI test for Rangerpicker.

### PR DESCRIPTION
**Pull request by dongsuwon**
Enact-DCO-1.0-Signed-off-by: Dongsu Won (dongsu.won@lgepartner.com)

### Checklist
* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
- Need to add ui tests for RangePicker.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Add ui tests for RangePicker

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]:# (Related issues, references)
WRP-5664
QWTC-2144
QWTC-2145
QWTC-2148
QWTC-2629

### CODE REVIEW

File: tests/ui/specs/RangePicker/RangePicker-RTL-specs.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		const negativeRangePicker = Page.components.rangePickerWithNegativeValues;
+		const rangePickerDisabled = Page.components.rangePickerDisabled;
+
+		it('should decrement to negative number [QWTC-2148]', async function () {
+			// Step 5: Click on the Left arrow button until '-5' displays.
+			await negativeRangePicker.decrementer(negativeRangePicker.self).click();
+			await browser.pause(500);
+			const newValue = extractValue(negativeRangePicker);
+			// Step 5 Verify: The minus sign displays to the Left of the number '5'.
+			expect(await newValue).to.equal(-5);
```
Comment (jeonghee27) [inline tests/ui/specs/RangePicker/RangePicker-RTL-specs.js:20]: I think screenshot test will be needed. 
for `The *minus* sign displays to the Left of the number '*5*'.`

I'll add

File: tests/ui/specs/RangePicker/RangePicker-RTL-specs.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+				await maxValueRangePicker.incrementer(maxValueRangePicker.self).click();
+				await maxValueRangePicker.incrementer(maxValueRangePicker.self).click();
+				await maxValueRangePicker.incrementer(maxValueRangePicker.self).click();
+				await browser.pause(500);
+				// Step 4 Verify:  Number 20 displays.
+				expect(await extractValue(maxValueRangePicker)).to.equal(20);
+
+				// Step 5: Click on Left arrow button 2 times.
+				await maxValueRangePicker.decrementer(maxValueRangePicker.self).click();
+				await maxValueRangePicker.decrementer(maxValueRangePicker.self).click();
```
Comment (jeonghee27) [inline tests/ui/specs/RangePicker/RangePicker-RTL-specs.js:45]: The "2145" TC is for verifying that the button drawn on the right side is the increase button and the button drawn on the left is the decrease button. Therefore, I don't think it's appropriate to click the increase button directly.
Could you change it to use the arrow keys to move the focus to the left button and press enter?

File: tests/ui/specs/RangePicker/RangePicker-RTL-specs.js (hunk 1 above)
Comment (dongsuwon) [reply tests/ui/specs/RangePicker/RangePicker-RTL-specs.js:20]: Thank you!

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Pull request merged**